rules/source-control-tools/rule.md

```diff
@@ -0,0 +1,143 @@
+
+The company behind GitKraken, Axosoft, has been recognized for its significant contributions to the developer community. For instance, Eric Amodio, the Chief Technology Officer at GitKraken, was awarded the Microsoft MVP Award for Developer Technologies in 2023. This award highlights his exceptional contributions to the developer community, particularly through GitLens, a popular Git extension for Visual Studio Code created by Amodio in 2016. GitLens has become a crucial tool in the daily coding processes of millions of developers, with its installation numbers surpassing 23 million.
+
+#### Pros:
+
+-   **Intuitive Interface:** GitKraken's graphical user interface is visually appealing and easy to understand, simplifying interactions with Git&#8203;
+-   **Integration with Platforms:** Smooth integration with GitHub, GitLab, and Bitbucket, enhancing workflow efficiency
+-   **Merge Conflict Editor:** Built-in tool for resolving merge conflicts, showcasing conflicting segments side-by-side
+-   **Drag-and-Drop Functionality:** Enables performing many Git operations easily with just drag-and-drop
+-   **Profile Management:** Useful for managing multiple profiles, distinguishing between personal and professional work
+-   **Dark and Light Themes:** Catering to various visual preferences
+-   **Extensive Documentation and Tutorials:** Assists new users in quickly becoming proficient with the tool
+-   **Cross-Platform Compatibility:** Available on Windows, MacOS, and Linux
+-   **Graphical Commit History:** Offers a vivid representation of repository history, enhancing the understanding of project flow
+-   **Powerful Search Functionality:** Makes finding commits, branches, and other elements straightforward
+
+#### Cons:
+
+-   **Performance Issues with Large Repositories:** Occasional slowdowns reported, particularly with substantial repositories
+-   **Freemium Model:** Advanced features require a subscription, despite a free version being available
+-   **No Git Credential Manager (GCM) Support:** Lacks GCM support, which can be a drawback for some users
+-   **Yearly Subscription Model:** Compared to one-off payment models, GitKraken adopts a yearly subscription approach
+-   **Marketing Aggressiveness:** Reports of excessive marketing emails after contacting support
+-   **App Stability Issues:** There have been instances of the app crashing upon opening in past versions, though this may have been resolved in newer updates
+-   **UI Aesthetics vs. Performance Trade-Off:** The attractive UI may come at the cost of slower performance compared to some alternatives
+-   **Limited Terminal and Workspaces Features:** GitKraken's terminal and workspaces might not meet the needs of users requiring advanced features
+
+![](https://www.linuxadictos.com/wp-content/uploads/gitkraken.png)
```

Comment:
Hey @PothieuG 

- [ ] 1. Please fix the syntax for caption. See https://ssw.com.au/rules/rule/#6-images
- [ ] 2. Make sure this image is sourced internally as per https://www.ssw.com.au/rules/images-should-be-hosted-internally/
- [ ] 3. Do the same for other image(s)